logic/standalone/conflict.py

```diff
@@ -0,0 +1,209 @@
+'''
+<SUMMARY TBA>
+
+USAGE EXAMPLE:
+    raw_dict = conflict.CheckConflicts(playdo, _LIST_LIGHTING_OBJ)
+    pruned_dict = conflict.PruneConflicts(playdo, conflict_dictionary)
+    conflict.FixConflicts(playdo, pruned_dict)
+'''
+
+import logic.common.log_utils as log
+import logic.common.tiled_utils as tiled_utils
+
+#--------------------------------------------------#
+'''Variables'''
+
+
+
+
+
+#--------------------------------------------------#
+'''Public Functions'''
+
+def CheckConflicts(playdo, list_scan_obj):
+	''' Task 1 - Check if there is any conflict '''
+	log.Info("\n--- Checking for sort value conflicts ---\n")
+
+	# Create the dictionary with each unique sort value as keys
+	dict = {}
+	for scanned_obj in list_scan_obj:
+		curr_obj_list = playdo.GetAllObjectsWithName(scanned_obj)
+		log.Info( f'# of \"{scanned_obj}\": {len(curr_obj_list)}' )
+		for obj in curr_obj_list:
+			curr_string = tiled_utils.GetSort(obj)
+			if curr_string == None: continue
```

Comment:
I actually was debating whether GetProperty should return None of an empty string. I know the "proper" way is to return None since there's nothing to return, and there *should* be a distinction from property simply does not need a value (e.g. `autoset` for BB).

So, returning a empty string is more like a band-aid fix, one that I want to hear if you have alternatives available, whereas the "continue if None is returned" here feels like the proper way to do things.

Reply:
As mentioned, I anticipate None to be for when the object does not contain the property, and an empty string to be for when the object does contain the property but with empty string as value. Would adding the comment plus an extra if-statement be more preferable?

```
			if curr_string == None: continue  # When object does not contain `_sort`
			if curr_string == '': continue    # When property has property but no value
```